README.md

```diff
@@ -1,74 +1,100 @@
-    - [ ] Check *Dismiss stale pull request approvals when new commits are pushed*
-    - [ ] Check *Require review from Code Owners*
-    - [ ] Check *Include Administrators*
-  - [ ] On the *Manage Access* tab add the appropriate groups
-- About Section (accessed on the main page of the repo, click the gear icon to edit)
-  - [ ] The repo should have a short description of what it is for
-  - [ ] Add one of the following topic tags:
-    | Topic Tag       | Usage                                    |
-    | --------------- | ---------------------------------------- |
-    | az              | For actions related to Azure             |
-    | code            | For actions related to building code     |
-    | certs           | For actions related to certificates      |
-    | db              | For actions related to databases         |
-    | git             | For actions related to Git               |
-    | iis             | For actions related to IIS               |
-    | microsoft-teams | For actions related to Microsoft Teams   |
-    | svc             | For actions related to Windows Services  |
-    | jira            | For actions related to Jira              |
-    | meta            | For actions related to running workflows |
-    | pagerduty       | For actions related to PagerDuty         |
-    | test            | For actions related to testing           |
-    | tf              | For actions related to Terraform         |
-  - [ ] Add any additional topics for an action if they apply    
-    
+# App Pool Status
+
+This action will return the status of an on-premises IIS app pool
+
+## Index <!-- omit in toc -->
+
+- [Inputs](#inputs)
+- [Outputs](#outputs)
+- [Prerequisites](#prerequisites)
+- [Example](#example)
+- [Code of Conduct](#code-of-conduct)
+- [License](#license)
 
 ## Inputs
-| Parameter | Is Required | Description           |
-| --------- | ----------- | --------------------- |
-| `input-1` | true        | Description goes here |
-| `input-2` | false       | Description goes here |
+
+| Parameter                  | Is Required | Description                          |
+| -------------------------- | ----------- | ------------------------------------ |
+| `server`                   | true        | The name of the target server        |
+| `app-pool-name`            | true        | IIS app pool name                    |
+| `service-account-id`       | true        | The service account name             |
+| `service-account-password` | true        | The service account password         |
+| `server-public-key`        | true        | Path to remote server public ssl key |
+
 
 ## Outputs
-| Output     | Description           |
-| ---------- | --------------------- |
-| `output-1` | Description goes here |
+
+| Parameter         | Description                    |
+| ----------------- | ------------------------------ |
+| `app-pool-status` | The status of the IIS app pool |
```

Comment:
It might be helpful to have another column indicating what the statuses could be so someone doesn't have to look it up in the documentation.